packages/govuk-frontend/src/govuk/components/task-list/task-list.yaml

```diff
@@ -0,0 +1,222 @@
+params:
+  - name: items
+    type: array
+    required: true
+    description: Array of item objects.
+    params:
+      - name: classes
+        type: string
+        required: false
+        description: Classes to add to the item `div`
+      - name: title.text
```

Comment:
Whilst we do use the parent.child notation for documenting objects and specs for arrays of objects elsewhere in govuk-frontend, this is complex enough that I'm finding it quite difficult to ingest. Could we please change it to be indented by object?